hera/editor/lib/updateUndoStack.tsx

```diff
@@ -1,9 +1,17 @@
+import Storage from '@deities/ui/Storage.tsx';
 import { EditorState, SetEditorStateFunction, UndoEntry } from '../Types.tsx';
 
+export const UNDO_STACK_KEY = (id: string | undefined) =>
+  `map-editor-undo-stack-${id ? `${id}` : 'fallback'}`;
+
+export const UNDO_STACK_INDEX_KEY = (id: string | undefined) =>
+  `map-editor-undo-stack-index-${id ? `${id}` : 'fallback'}`;
+
```

Comment:
Since these are functions they should be `getUndoStackKey` and `getUndoStackKeyIndex`.